playbooks/openshift-node/private/clean_image.yml

```diff
@@ -27,3 +27,41 @@
     file:
       state: absent
       path: "/var/lib/cloud/"
+
+- name: Set gquota for slash filesystem
```

Comment:
No need for a new play here, we're operating on the same hosts as the previous play.  Combine.